libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -109,25 +117,113 @@ describe("Drilling", () => {
+                firstWidget.waitTableLoaded().getTable().hasCellValue(index, 0, value);
+            });
+
+            firstWidget.waitTableLoaded().getTable().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should("have.text", `table has row column measure › ` + DIRECT_SALES);
+            productNameValues.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.getTable().waitLoaded().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `table has row column measure › ` + DIRECT_SALES + " › " + `${productNameValues[0]}`,
+                );
+            YearList.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.back();
+            drillModal.getTable().waitLoaded().click(1, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `table has row column measure › ` + DIRECT_SALES + " › " + `${productNameValues[1]}`,
+                );
+            YearListClose.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.close();
+            api.setUpDrillDownAttribute(DISPLAYFORM_PRODUCT);
+        });
+
+        it("Drilling down on a table that has two drillable attributes separate", () => {
+            const assertDrillToQuarterValue = ["Q2/2010"];
+
+            Navigation.visit("dashboard/implicit-drill");
+            api.setUpDrillDownAttribute(DISPLAYFORM_PRODUCT, YEAR_CLOSE);
+            firstWidget.waitTableLoaded().getTable().click(0, 0);
+            drillModal.getTitleElement().should("have.text", `Table with years › ` + YEAR_2010);
+            drillModal.getTable().waitLoaded().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `Table with years › ` + YEAR_2010 + ` › ` + `${assertDrillToQuarterValue[0]}`,
+                );
+            drillModal.getTable().hasCellValue(0, 0, "Jun 2010");
+            drillModal.close();
+            firstWidget.waitTableLoaded().getTable().click(0, 1);
+            drillModal.getTitleElement().should("have.text", `Table with years › ` + DIRECT_SALES);
+            productNameValues.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.getTable().waitLoaded().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `Table with years › ` + DIRECT_SALES + " › " + `${productNameValues[0]}`,
```

Comment:
don't use index of list like that` ${productNameValues[0]}`, please make it clearly by inputing correct value. Please help check other places